### PR TITLE
DR2-1604 Update asset row with ingested_PS

### PR DESCRIPTION
**Pull request by MancunianSam**
This is a step function and policy change to update the ingested_PS
attribute on the asset row in Dynamo when the reconcile is successful.

### CODE REVIEW

**Review by techncl — COMMENTED** [review]
Looks good! Just two comments

File: templates/sfn/ingest_sfn_definition.json.tpl [hunk 1]
```diff
@@ -462,13 +462,28 @@
...
+              "TableName": "${ingest_files_table_name}",
+              "Key": {
+                "id": {
+                  "S.$": "$.reconciliationSnsMessage.assetId"
+                }
+              },
+              "UpdateExpression": "SET ingested_PS = :ingestedPSValue",
+              "ExpressionAttributeValues": {
+                ":ingestedPSValue": {
+                  "S": "true"
```
[the remark this answers is not in the document]
Comment (MancunianSam) [reply templates/sfn/ingest_sfn_definition.json.tpl:483]: The value isn't boolean in Dynamo. It's been set up as a string and I didn't want to go messing with it. If you try BOOL, you get an error

[the remark this answers is not in the document]
Comment (techncl) [reply templates/sfn/ingest_sfn_definition.json.tpl:483]: Ah, I see

**Review by techncl — APPROVED** [review]
👍

**Pull request merged**